### PR TITLE
IDV-1633: disable card scanner by default in rn

### PR DESCRIPTION
**Pull request by valllllll2000**
Task https://fidelapi.atlassian.net/browse/IDV-1633

Card scanner disabled by default

To test use the Fidel setup options `enableCardScanner`: true or false inside `example/App.js`

On Android you can test both combinations of card scanner and autoscan on the emulator but you cannot actually scan cards. On iOS you can only test autoscan (on the simulator card scanning is not possible). It you want to actually test the scanning of cards itself, you will need a real device.

### CODE REVIEW

File: ios/Adapters/FidelSetupAdapter.swift [hunk 1]
```diff
@@ -41,7 +41,7 @@ class FidelSetupAdapter: NSObject {
         }
     }
     
-    func setOptions(_ options: [String: Any?]) {
+    func setOptions(_ options: [String: Any?]) {
```
Comment (valllllll2000) [inline ios/Adapters/FidelSetupAdapter.swift:44]:
```suggestion
    func setOptions(_ options: [String: Any?]) {
```

File: android/src/test/java/com/fidelreactlibrary/FidelSetupAdapterTests.java [hunk 2]
```diff
@@ -790,6 +792,45 @@ public void test_IfDoesHaveProgramTypeKey_ShouldSetTheValueProvidedByTheProgramT
         assertEquals(ProgramType.TRANSACTION_STREAM, Fidel.programType);
     }
 
+    @Test
+    public void test_IfDoesNotHaveEnableCardScannerKey_DoNotSetThisPropertyForFidel() {
+        ReadableMapStub map = ReadableMapStub.withoutOptionsKey(FidelSetupKeys.Options.ENABLE_CARD_SCANNER);
+        ReadableMapStub optionsMap = (ReadableMapStub) map.mapsForKeysToReturn.get(FidelSetupKeys.OPTIONS.jsName());
+        assertNotNull(optionsMap);
+
+        Fidel.enableCardScanner = true;
```
Comment (MariaJoaoMiraPaulo) [inline android/src/test/java/com/fidelreactlibrary/FidelSetupAdapterTests.java:801]: Why are we doing this and then checking that's true? 

Wouldn't this be enough to test what we have in the description? 
```
assertFalse(optionsMap.keyNamesAskedValueFor.contains(FidelSetupKeys.Options.ENABLE_CARD_SCANNER.jsName()));
```

Comment (valllllll2000) [reply android/src/test/java/com/fidelreactlibrary/FidelSetupAdapterTests.java:801]: I copied it from another test but basically we want to check the property is set correctly. The optionsMap is a stub object created for testing purposes.

Comment (MariaJoaoMiraPaulo) [reply android/src/test/java/com/fidelreactlibrary/FidelSetupAdapterTests.java:801]: For the following tests, it makes sense to assert the `Fidel.enableCardScanner`, but for this one, I am not sure it does. But not a blocker on my side 😄 I let you decide!

Comment (MariaJoaoMiraPaulo) [reply android/src/test/java/com/fidelreactlibrary/FidelSetupAdapterTests.java:801]: Got it 😄

**Review by Axeirt — APPROVED** [review]

**Pull request merged**